pkg/abi/linux/msgqueue.go

```diff
@@ -47,7 +47,7 @@ const (
 	MSGSSZ  = 16
 
 	// MSGSEG is simplified due to the inexistance of a ternary operator.
-	MSGSEG = (MSGPOOL * 1024) / MSGSSZ
+	MSGSEG = 0xffff
```

Comment:
Could you explain why this needs to change? Was it wrong before?

Reply:
`MSGSEG` is defined within the kernel as
```
#define __MSGSEG ((MSGPOOL * 1024) / MSGSSZ) /* max no. of segments */
#define MSGSEG (__MSGSEG <= 0xffff ? __MSGSEG : 0xffff)
```
Given that this is not doable in Go, and the value is unused within the kernel, I chose to simplify it. This produced a problem when implementing `msgctl(IPC_INFO)`, because the value of `MSGSEG` didn't fit into `MsgInfo.MsgSeg` which is a `uint16`, so I changed it to fit.